media-libs/mesa/mesa-9999.ebuild

```diff
@@ -127,6 +143,7 @@ DEPEND="${RDEPEND}
 		x11-base/xorg-proto
 	)
 "
+#Meson regression nouveau/NVK (upstream bug #10855)
```

Comment:
_Hello @MrDuartePT,_

I believe it's worth providing direct links for better transparency.
```suggestion
# meson-1.4.0 contains a regression, so it fails to compile nouveau/NVK
# see https://gitlab.freedesktop.org/mesa/mesa/-/issues/10855
```